### PR TITLE
Implemented .env file API key storage

### PR DESCRIPTION
**Pull request by chriscarrollsmith**
.env files are the standard way to securely store API keys in Python projects. README explains how to use the .env file (rather than the .txt file use in the previous workflow). I also updated .gitignore.

### CODE REVIEW

**Comment by ra312** [comment]
can you please provide description ?

**Comment by chriscarrollsmith** [comment]
Edited the description above.

**Comment by ra312** [comment]
it could be better to turn on GitGuardian, just in case

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY=your_api_key
```
Comment (biobootloader) [inline .env.sample:1]:
```suggestion
OPENAI_API_KEY=your_api_key

```

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 venv
 openai_key.txt
```
Comment (biobootloader) [inline .gitignore:2]:
```suggestion
```

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,4 @@
...
+.venv
+.env
```
Comment (biobootloader) [inline .gitignore:4]:
```suggestion
.env

```

**Review by biobootloader — COMMENTED** [review]
Thank you! Can you also add `python-dotenv` to the `requirements.txt`?

**Comment by chriscarrollsmith** [comment]
Done!

**Pull request merged**